README.md

```diff
@@ -1,2 +1,109 @@
-# truly_automated_tests
+# Truly automated tests example
 Sample project for "Truly automated testing" concept demo
+
+
+## API  Spec
+
+### Methods
+
+ - **/endpoint/**
+    - **GET** - List all objects. Will be called **LIST** in further mentions
+    - **POST** - Create an object. Will be called **POST** in further mentions
+ - **/endpoint/\<obj_id\>/**
+    - **GET** - Get exact object data. Will be called **GET** in further mentions
```

Comment:
What about `delete` method?

Reply:
This will be a demo app. So it looks reasonable to keep it small but still representative.